indexer/packages/postgres/__tests__/stores/wallet-table.test.ts

```diff
@@ -113,45 +111,12 @@ describe('Wallet store', () => {
         firstFillTime.toISO(), // exclusive -> filters out first fill from each subaccount
         firstFillTime.plus({ minutes: 2 }).toISO(),
       );
-      wallet = await WalletTable.findById(defaultWallet.address);
-      expect(wallet).toEqual(expect.objectContaining({
+      const wallet2 = await WalletTable.findById(defaultWallet.address);
+      expect(wallet2).toEqual(expect.objectContaining({
         ...defaultWallet,
         totalVolume: '105', // 103 + 2
       }));
     });
-
-    it('Successfully upserts persistent cache', async () => {
```

Comment:
why delete this test?

Reply:
The query to update total volume was modified to not update the persistent cache. The roundtable task calls persistent cache upsert within the same transaction as total volume update.

This pattern is what the affiliate info uses.

Reply:
Persistent cache updating is checked in the roundtable task unit tests.